go-apps/meepctl/cmd/delete.go

```diff
@@ -146,11 +146,13 @@ func deleteDep(cobraCmd *cobra.Command) {
 	go k8sDeletePvc("data-meep-elasticsearch-data-0", cobraCmd, messages)
 	go k8sDeletePvc("data-meep-elasticsearch-master-0", cobraCmd, messages)
 	go k8sDeletePvc("data-meep-elasticsearch-master-1", cobraCmd, messages)
+        go k8sDeletePvc("meep-influxdb-data-meep-influxdb-0", cobraCmd, messages)
```

Comment:
You should not need this. It seems you are missing a PVC in your chart templates.

Reply:
removed.. now handled as part of a new pvc yaml file